starlette/requests.py

```diff
@@ -278,9 +285,15 @@ async def _get_form(self, *, max_files: int | float = 1000, max_fields: int | fl
         return self._form
 
     def form(
-        self, *, max_files: int | float = 1000, max_fields: int | float = 1000
+        self,
+        *,
+        max_files: int | float = 1000,
+        max_fields: int | float = 1000,
+        max_part_size: int | float = 1024 * 1024,
```

Comment:
This doesn't need to be a float.

Reply:
Ah, i just followed other params. Will change them all.

Reply:
No, the others allow float.

Reply:
How'd files and fields be fraction? Shouldn't it be a whole number always? I'm just curious.

Reply:
Because they accept `float("inf")`.